.travis.yml

```diff
@@ -1,18 +1,25 @@
 language: ruby
+dist: trusty
 
 rvm:
-  - 2.2
+  # - "2.2"
+  - "2.3.1"
 
-branches:
-  only:
-  - master
-  - develop
-  - deploy-release
-  - /^issue.*$/
+gemfile:
+  - cf.Gemfile
+
+# branches:
```

Comment:
Is this supposed to be commented out or is that left over from debugging? Seems better to remove entirely if it's not needed.